publish.py

```diff
@@ -65,7 +65,7 @@ def main():
         subprocess.check_call(["git", "fetch"])
         our_hash = subprocess.check_output(["git", "rev-parse", "HEAD"])
         upstream_hash = subprocess.check_output(["git", "rev-parse", "@{u}"])
-        assert our_hash == upstream_hash, "local and/or remote are missing commits"
+#        assert our_hash == upstream_hash, "local and/or remote are missing commits"
```

Comment:
supposed to be in here?

Reply:
Ah, no, I forgot to take that out.  Will correct.